test/stage2_fwd.jl

```diff
@@ -6,36 +6,34 @@ module stage2_fwd
         @test sin′(1.0) == cos(1.0)
     end
     let sin′′ = Diffractor.dontuse_nth_order_forward_stage2(Tuple{typeof(mysin), Float64}, 2)
-        # This broke some time between 1.10 and 1.11-DEV.10001
-        @test isa(sin′′, Core.OpaqueClosure{Tuple{Float64}, Float64}) broken=VERSION>=v"1.11-"
```

Comment:
oh yay!

Reply:
I don't know too much why these broken test cases have been fixed, but I guess it was probably because the previous `(::∂☆recurse)(...)` implementation has the serious flaw (i.e. the generator uses function that hasn't been defined at the time when the generator function is defined)?